packages/cra-template/template.json

```diff
@@ -58,12 +41,29 @@
       "excludeAfterRemap": true
     },
     "devDependencies": {
+      "@cypress/code-coverage": "3.9.12",
+      "@nimblehq/eslint-config-nimble-react": "^1.0.0",
+      "@nimblehq/stylelint-config-nimble": "^1.0.0",
+      "@testing-library/cypress": "8.0.2",
+      "@testing-library/jest-dom": "5.16.4",
+      "@testing-library/react": "13.1.1",
+      "@testing-library/user-event": "14.1.1",
+      "@types/jest": "27.4.1",
+      "@types/node": "17.0.21",
+      "@types/react": "17.0.40",
+      "@types/react-dom": "17.0.13",
       "@cypress/instrument-cra": "1.4.0",
+      "cypress": "9.5.2",
+      "cypress-react-selector": "2.3.16",
       "danger": "10.9.0",
       "danger-plugin-code-coverage": "1.1.9",
       "danger-plugin-istanbul-coverage": "1.6.2",
       "danger-plugin-jest-codecov": "0.0.8",
```

Comment:
Seeing from our dangerfile, we're only using `istanbul-coverage` plugin. So I'll also remove other unused dependencies

```js
// Our danger file
import { istanbulCoverage } from 'danger-plugin-istanbul-coverage';

schedule(
  istanbulCoverage({
  ...
```